app/src/Outlander/Handlers/GotoCommandHandler.swift

```diff
@@ -64,6 +64,15 @@ class GotoComandHandler: ICommandHandler {
                 : $0
         }.joined(separator: " ")
 
+        // what if I saved \(args) to $mapperpath global var? ~DAH
+        // fill mapperpath with list|of|moves then save to global
+        let mapperpath = moves.map {
+            $0.range(of: " ") != nil
+                ? "\"\($0)\""
+                : $0
+        }.joined(separator: "|")
+        context.globalVars["mapperpath"] = mapperpath
```

Comment:
I think creating a list with `|` makes sense because then you could use the list functions in the scripting language to deal with it easier.

This duplicates the "map (filter)" code with `moves`. This can be simplified by splitting the initial results and doing different joins with just the separators.

```swift
let argList = moves.map {
  $0.range(of: " ") != nil
      ? "\"\($0)\""
       : $0
}

let args = argList.joined(separator: " ")
let mapperpath = argList.joined(separator: "|")
```